src/core/common/message.cpp

```diff
@@ -148,6 +149,9 @@ ThreadError Message::ResizeMessage(uint16_t aLength)
     FreeBuffers(curBuffer);
 
     return kThreadError_None;
+
+exit:
```

Comment:
Code style: we prefer to have a single return point in a given function.  Just define a local variable for the error and set it in the `VerifyOrExit()`.